src/app/cmdoptions.cpp

```diff
@@ -422,7 +422,7 @@ QBtCommandLineParameters parseCommandLine(const QStringList &args)
             else if (arg == NO_SPLASH_OPTION) {
                 result.noSplash = true;
             }
-#else
+#elif defined(DISABLE_GUI) && !defined(Q_OS_WIN)
```

Comment:
here too

Reply:
the logic is:
Q_OS_WIN && DISABLE_GUI: print version in console, disable daemonization
Q_OS_WIN && !DISABLE_GUI: display version in a message box, disable daemonization
!Q_OS_WIN && DISABLE_GUI : print version in console, enable daemonization
!Q_OS_WIN && !DISABLE_GUI: print version in console, disable daemonization

Reply:
@knackebrot 
You missed this one, it can be written as `#elif !defined(Q_OS_WIN)`

Reply:
fixed